pkg/apiserver/profile/profile.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package profile
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+
+	"github.com/pkg/errors"
+)
+
+// the default time interval of profiling.
+const grabInterval = 30
+
+func fetchSvg(ctx context.Context, t, addr, filePrefix string) (string, error) {
```

Comment:
maybe better to call it `fetchFlameGraphSVG`.

Reply:
go based components are not the FlameGraph instead

Reply:
oh. yes. It is a dot graph..